style.css

```diff
@@ -9,6 +9,93 @@ body {
   font-family: Poppins, sans-serif;
 }
 
+.mobile-header {
+  background-color: #fff;
+}
+
+.logo {
+  color: #6070ff;
+  font-weight: 700;
+  font-family: Poppins, sans-serif;
+}
+
+.openMenu {
+  display: block;
+  color: #6070ff;
+}
+
+.closeMenu {
+  color: #fff;
+}
+
+nav .navbar {
+  display: flex;
+  justify-content: space-between;
+}
+
+nav .mainMenu {
+  list-style: none;
+  height: 100vh;
+  position: fixed;
+  padding-top: calc(40% - 24px / 2);
+  text-align: start;
+  right: 0;
+  left: 0;
+  z-index: 10;
+  gap: 0;
+  flex-direction: column;
+  justify-content: flex-start;
+  background-color: #6070ff;
+  opacity: 0.9;
+  display: none;
+  transition: top 1s ease;
+}
```

Comment:
- [ Optional ] Great job implementing the changes from the previous reviewer, I think your design will improve greatly if you added a `blur` effect on the `mobile menu` when it's opened so as to match the Figma design. Please note that on the Figma design it's very difficult to see the content in the `background` when the mobile menu is open, this is because of the `blur` effect. I would like to suggest that you refactor this part of your CSS as follows.

```CSS
nav .mainMenu {
  list-style: none;
  height: 100vh;
  position: fixed;
  padding-top: calc(40% - 24px / 2);
  text-align: start;
  right: 0;
  left: 0;
  z-index: 10;
  gap: 0;
  flex-direction: column;
  justify-content: flex-start;
  display: none;
  background-color: rgba(50, 70, 255, 0.8);
  backdrop-filter: blur(5.5px);
  transition: top 1s ease;
}
```

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/223690617-c7d755d0-4436-4ac2-9dc4-c3f560cecc80.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/223690927-aafcd7b4-c369-4a64-8fcd-cc55632ba937.png" width="375px" />